inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/include/reshape_dims.cl

```diff
@@ -125,6 +125,32 @@ inline uint8 FUNC(reshape_6_to_4)(uint o, uint i, uint w, uint z, uint y, uint x
     return (uint8)(0, dst_b, dst_f, 0, 0, dst_y, dst_x, 0);
 }
 
+inline uint8 FUNC(reshape_6_to_5)(uint o, uint i, uint w, uint z, uint y, uint x,
```

Comment:
nit: please update year in copyright message for modified files

Reply:
Done